### DISCUSSION TITLE
Browser fails to load next application

### DISCUSSION THREAD

**Discussion by HerbertNtim** [vercel/next.js#65648] [Help]
### Summary

![Screenshot (245)](https://github.com/vercel/next.js/assets/104742927/e34777e2-32f7-4b3f-b75c-0ce5446c02a4)

**Comment by icyJoseph** [comment]
Show us the logs in the browser console.

**Reply by HerbertNtim** [reply]
I have provided the errors from the console. Help me out please.
This happens to every browser I open with the application

**Reply by icyJoseph** [reply]
I see, I've never been able to help with that issue. It seems to happen mostly on Windows, but let's try.

First, what Node.js version do you have? 

Second, what's the path to the project?

Also, perhaps, Next.js version?

**Reply by HerbertNtim** [reply]
the node version is v21.6.2 and i am the latest version of next js

**Comment by HerbertNtim** [comment]
These are errors from the console
![Screenshot (246)](https://github.com/vercel/next.js/assets/104742927/3a404afa-6aa1-4e4b-b248-f0585e306bec)
![Screenshot (247)](https://github.com/vercel/next.js/assets/104742927/d7bd87c0-31cd-478d-b2b2-5d9428dd9339)

**Comment by HerbertNtim** [comment]
I solved it by updating my windows and restarting my project again

**Reply by icyJoseph** [reply]
Hi, then please close the discussion :) good that you got it to work.

**Reply by HerbertNtim** [reply]
I have encounted the same error again.
